libcodechecker/libhandlers/analyze.py

```diff
@@ -127,16 +127,24 @@ def add_arguments_to_parser(parser):
                         dest="compiler_includes_file",
                         required=False,
                         default=None,
-                        help="Read the compiler includes from the specified "
-                             "file rather than invoke the compiler "
+                        help="DEPRECATED. Read the compiler includes from the "
```

Comment:
Deprecated arguments used to have a wrapper somewhere which allowed setting them to behave with emitting a warning that a deprecated argument is used. I think this should be introduced again, or just simply the argument parser round (`__handle`) written in a way that the user is warned about the deprecated argument.

Reply:
I'll try to search this wrapper in the git history. However, I'd propose a new policy rule: if there are such changes which involve other parts of the source code requiring their change, and not in the scope of the current pull request, then a new issue should be registered, maybe by additionally providing the assignee. What do you think?

Reply:
https://github.com/Ericsson/codechecker/blob/451573065bac5ba8008674817c6e66e7bb98cd7d/libcodechecker/libhandlers/check.py#L56